langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -104,11 +137,9 @@ public isolated function filter(Type[] arr, @isolatedParam function(Type val) re
 # The combining function takes the combined value so far and a member of the array,
 # and returns a new combined value.
 #
-# For example
 # ```
-# reduce([1, 2, 3], function (int total, int n) returns int { return total + n; }, 0)
+# [1, 2, 3].reduce(isolated function (int total, int next) returns int => total + next, 0) ⇒ 6
```

Comment:
```suggestion
# [1, 2, 3].reduce(isolated function (int accTot, int next) returns int => accTot + next, 0) ⇒ 6
```
This is the accumulated total. So how about using `accTot` instead of `total`?

Reply:
How is accTotal different from total?